Cargo.toml

```diff
@@ -9,7 +9,8 @@ homepage = "https://clickhouse.com"
 license = "MIT OR Apache-2.0"
 readme = "README.md"
 edition = "2021"
-rust-version = "1.67.0" # update `derive/Cargo.toml` and CI if changed
+# update `derive/Cargo.toml` and CI if changed
+rust-version = "1.67.0"
```

Comment:
I think we can raise this to 1.70 to match hyper-rustls too

Reply:
Yep it seems they bumped it in a minor release, which isn't nice